test/evergreen.yml

```diff
@@ -558,6 +558,15 @@ variables:
         vars:
           format_test_script_args: -e "SEGFAULT_SIGNALS=all" -b "catchsegv ./t" -t 360
 
+  - &format-abort-recovery-stress-test
```

Comment:
I feel it's a bit overdesign by using yaml anchor "&" as it's only been referenced by a single task below. A vanilla task should do for now.

Reply:
See 7d3b047af45405f8727de69a509c88344b9566b9